docs/tutorials/intro.md

```diff
@@ -5,35 +5,35 @@ sidebar_position: 1
 
 # Introduction
 
-There are a number of ways for developers to interact with the MXC ecosystem. These forms of participation are integral to the further development and deployment of the MXProtocol.
+Developers can engage with the MXC ecosystem in a variety of ways. These forms of participation are integral to the further development and deployment of the MXProtocol.
 
 ## Direct Participation
 
 ### Beta Testing
-Beta testing new features before they are released is a great way to help us improve the overall user experience with our products. Here are beta opportunities currently available:
+We can greatly enhance the overall user experience with our products by beta testing new features prior to their release. The following are the current beta openings:
 
 | Product | Beta Type | How to Participate |
 | ------- | --------- | ------------------ |
 | DataDash | Closed | [Instructions](tutorials/datadash/beta.md) |
 | MXC Controller | Closed | *Coming Soon* |
 | M2 Pro (MatchX) | Open | *Coming Soon* |
 
-*A beta version is pre-release and considered unstable. This means that you may experience some issues. Normally these will be minimal, however participating in an M2 Pro beta could potentially affect your mining. MXC Controller and DataDash betas will not have any influence on your overall mining potential.*
+*A beta version is pre-release and considered unstable. This implies that you may experience some issues. Normally these issues will be minimal, (????) however participating in an M2 Pro beta could potentially affect your mining. MXC Controller and DataDash betas will not have any influence on your overall mining potential.*
```

Comment:
what is ????

Reply:
any ???? is things that I wanted to check with you. so I flagged it so I can get back to it